frontend/packages/dev-console/integration-tests/features/web-terminal/web-teminal-basic.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Web Terminal
```

Comment:
Web terminal in developer perspective

Reply:
Web Terminal can be opened in any perspective. That's I have kept it like that.